website/docs/components/code-editor/partials/code/component-api.md

```diff
@@ -0,0 +1,57 @@
+## Component API
+
+This component uses [CodeMirror](https://codemirror.net/) under the hood.
```

Comment:
Suggestion: should we explicitly say what version of CodeMirror we use? esp since the previous version that most teams are using is super different from ours?